src/pages/create-index.html

```diff
@@ -1,16 +1,7 @@
-<link href="css/lib/angular-xeditable/xeditable.min.css?v=[AIV]{version}[/AIV]" rel="stylesheet">
```

Comment:
Is this obsolete already regarding your changes or it was already redundant dependency?

Reply:
Yes, it was redundant dependency.